### ISSUE TITLE
BasicUI issue with updating default slider for dimmer item

### ISSUE THREAD

**Issue by mherwege** [eclipse/smarthome#3476]
Issue #3036 has only been partially solved.

This is my test sitemap fragment:
```
Frame {
	Switch item=Licht_Zithoek
 	Slider item=Licht_Zithoek
 	Setpoint item=Licht_Zithoek minValue=0 maxValue=100 step=5
	Default item=Licht_Zithoek	
}
```
Licht_Zithoek is a dimmer item. Updating any of the first 3 updates the top 3 immediately, but not the 4th (default item). Updating the default item will update the other 3 for me, but that may be because of the binding that I am using that gives immediate feedback from the physical device. So I am not sure if is reacting to the default slider update or the binding feedback sending an update to the item.

**Comment by lolodomo** [comment]
Your issue is relative to a general issue with default widgets.
I will test your case but I am almost sure that my PR #3471 will fix this case too.

**Comment by lolodomo** [comment]
@sjka @kaikreuzer  : please close this issue, the merged PR #3471 fixed it.
